core/common/src/main/java/alluxio/worker/block/CreateBlockOptions.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.block;
+
+import alluxio.annotation.PublicApi;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * Method options for create a block.
+ */
+@PublicApi
+@NotThreadSafe
+public final class CreateBlockOptions {
+  private String mAlluxioPath;
+  private long mInitialBytes;
+  private String mMedium;
+
+  /**
+   * Constructor.
+   *
+   * @param alluxioPath the alluxio path
+   * @param medium the medium
+   * @param initialBytes the initialBytes
+   */
+  public CreateBlockOptions(String alluxioPath, String medium, long initialBytes) {
+    mAlluxioPath = alluxioPath;
+    mMedium = medium;
+    mInitialBytes = initialBytes;
+  }
+
+  /**
+   * @return the alluxio path
+   */
+  public String getAlluxioPath() {
+    return mAlluxioPath;
+  }
+
+  /**
+   * @return the initial bytes
+   */
+  public long getInitialBytes() {
```

Comment:
or document the default value if not set

Reply:
this is an long, must to have